zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/TopologyController.java

```diff
@@ -47,7 +47,7 @@ public TopologyResponse get() {
           .clusterSize(topology.getClusterSize())
           .partitionsCount(topology.getPartitionsCount())
           .replicationFactor(topology.getReplicationFactor())
-          .lastCompletedChangeId(topology.getLastCompletedChangeId());
+          .lastCompletedChangeId(String.valueOf(topology.getLastCompletedChangeId()));
```

Comment:
🙃 Could be beneficial to use `KeyUtil.keyToString` here, like other controllers do.